src/portlet/Portlet.tsx

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import * as React from "react";
+import {classNames} from "../utils";
+import {Icon, Direction} from "../icon";
+
+type PortletProps = {
+    style?: any;
+    className?: string;
+    header: any;
+};
+
+type PortletState = {
+    toggleContent: boolean;
+    iconStyle: any;
+};
+
+export class Portlet extends React.PureComponent<PortletProps, PortletState> {
+    state: PortletState = {
+        toggleContent: true,
+        iconStyle: {
+            transition: "all .3s ease-in 0s",
+            float: "right",
+            color: "#007DB8",
+        },
+    };
+    handleClick = () => {
+        const {toggleContent} = this.state;
+        this.setState({toggleContent: !toggleContent});
+        toggleContent
+            ? this.setState({
+                  iconStyle: {
+                      transition: "all .3s ease-in 0s",
+                      transform: "rotate(180deg)",
+                      float: "right",
+                      color: "#007DB8",
+                  },
+              })
+            : this.setState({
+                  iconStyle: {
+                      transition: "all .3s ease-in 0s",
+                      float: "right",
+                      color: "#007DB8",
+                  },
+              });
+    };
+
+    render() {
+        const {className, style, header, children} = this.props;
```

Comment:
Where are we using 'style' prop? Is it applicable to the children or the main'div'?

Reply:
Right now we are not using it but we may need it when we will implement this component in Vsphere as we dont have vmware styling here.

Reply:
Addressed - removed style props